content/docs/k8s/reference.md

```diff
@@ -368,7 +382,7 @@ IdentityProvider see https://www.pomerium.com/docs/identity-providers/
                 </p>
                 <p>
                     
-                    Scopes correspond to access <a href="https://www.pomerium.com/reference/#identity-provider-scopes">privilege scopes</a>.
+                    Scopes is a list of <a href="https://www.pomerium.com/reference/#identity-provider-scopes">privilege scopes</a> to request access to from the Identity Provider.
```

Comment:
This one still redirects to Reference homepage. Remove the "#" from the URL and it takes you to the correct page.